crates/forge/tests/it/invariant.rs

```diff
@@ -165,142 +165,6 @@ async fn test_invariant_filters() {
                     None,
                 )],
             ),
-            (
-                "default/fuzz/invariant/common/InvariantShrinkWithAssert.t.sol:InvariantShrinkWithAssert",
```

Comment:
Do we have regression here?

Reply:
🤷  https://github.com/foundry-rs/foundry/pull/8459